tasks/build.yml

```diff
@@ -0,0 +1,83 @@
+---
+
+- name: Create Prometheus MQTT Exporter directories
+  become: yes
+  file:
+    path: "{{ item }}"
+    state: directory
+    owner: "{{ prometheus_mqtt_exporter.user }}"
+    group: "{{ prometheus_mqtt_exporter.group }}"
+    mode: 0750
+  with_items:
+    - "{{ prometheus_mqtt_exporter.path.install }}"
+    - "{{ prometheus_mqtt_exporter.path.install }}/bin"
+    - "{{ prometheus_mqtt_exporter.path.install }}/prometheus_mqtt_exporter"
+    - "{{ prometheus_mqtt_exporter.path.log }}"
+
+- name: Copy wrapper shell script
+  become: yes
+  copy:
+    dest: "{{ prometheus_mqtt_exporter.path.install }}/bin/prometheus_wrapper"
+    group: "{{ prometheus_mqtt_exporter.group }}"
+    mode: 0755
+    owner: "{{ prometheus_mqtt_exporter.user }}"
+    src: prometheus_wrapper
+
+- name: Deploy config from template
+  become: yes
+  template:
+    dest: "{{ prometheus_mqtt_exporter.path.install }}/prometheus_mqtt_exporter/config.yml"
+    group: "{{ prometheus_mqtt_exporter.group }}"
+    mode: 0644
+    owner: "{{ prometheus_mqtt_exporter.user }}"
+    src: config.yml.j2
```

Comment:
This will need a handler to restart the service on changes to this file as well.